src/dmd/frontend.h

```diff
@@ -829,7 +829,7 @@ struct Global final
     FileManager* fileManager;
     enum : int32_t { recursionLimit = 500 };
 
-    FileName(*preprocess)(FileName );
+    FileName(*preprocess)(FileName , bool );
```

Comment:
Looks like the header generator doesn't handle `out` cc @MoonlightSentinel

Reply:
https://github.com/dlang/dmd/pull/14093